### ISSUE TITLE
Fatal error:  Unknown profile version

### ISSUE THREAD

**Issue by januszoles** [laurent22/joplin#10477]
### Operating system

macOS

### Joplin version

2.14.22

### Desktop version info

Joplin version: 2.14.22 (mac os intel)

### Current behaviour

Joplin not responding
Unknown profile version. Most likely this is an old version of Joplin, while the profile was created by a newer version. Please upgrade Joplin at https://joplinapp.org and try again.
Joplin version: 2.14.22
Profile version: 47
Expected version: 46

Error: Unknown profile version. Most likely this is an old version of Joplin, while the profile was created by a newer version. Please upgrade Joplin at https://joplinapp.org and try again.
Joplin version: 2.14.22
Profile version: 47
Expected version: 46
at JoplinDatabase.<anonymous> (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:351:23)
at Generator.next (<anonymous>)
at /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:8:71
at new Promise (<anonymous>)
at __awaiter (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:4:12)
at JoplinDatabase.upgradeDatabase (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:331:16)
at JoplinDatabase.<anonymous> (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:909:43)
at Generator.next (<anonymous>)
at fulfilled (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:5:58)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by bamboleate** [comment]
I have the same issue with the . It all began with an update I couldn't download as there was no intel-mac-app, but the alert bugged me and so I tried via https://joplinapp.org/download/, I tried via brew, the newest version is definetely newer than the one I had installed, and my problem as well is it should use ver 46 while my "profile version" seems to be ver 47. 
It'd be nice to know my previous version number, so I could just revert.
Or any other solution... 

I have tried as you can see ver 2.14.16 now, but it's the same for 2.14.20 and 2.14.22 as well, and 2.14.22 seems to be the newest version?!

---message---

An error occurred: Fatal error:

Unknown profile version. Most likely this is an old version of Joplin, while the profile was created by a newer version. Please upgrade Joplin at https://joplinapp.org and try again.
Joplin version: 2.14.16
Profile version: 47
Expected version: 46

Error: Unknown profile version. Most likely this is an old version of Joplin, while the profile was created by a newer version. Please upgrade Joplin at https://joplinapp.org and try again.
Joplin version: 2.14.16
Profile version: 47
Expected version: 46
at JoplinDatabase.<anonymous> (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:351:23)
at Generator.next (<anonymous>)
at /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:8:71
at new Promise (<anonymous>)
at __awaiter (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:4:12)
at JoplinDatabase.upgradeDatabase (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:331:16)
at JoplinDatabase.<anonymous> (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:909:43)
at Generator.next (<anonymous>)
at fulfilled (/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/JoplinDatabase.js:5:58)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by bamboleate** [comment]
Sooo, I have managed to install ver 3.0.6 on an iMac 2017 and Joplin works again. Updaters, infos and version numbers are a little messed up imho. So: [GO HERE](https://github.com/laurent22/joplin/releases/tag/v3.0.3) and download.

Man it took forever to find out. I apreciate the devs putting all time into dev'ing, but a little tidy linkage would have served us well.. 

I still wonder how I managed to install "profile version 47", but I won't get all that back I guess. So. Good luck to anyone who had the same issue/landed here.
